examples/01_random_field/07_precipitation.py

```diff
@@ -0,0 +1,114 @@
+"""
+Creating Synthetic Precipitation Fields
+---------------------------------------
+
+In this example we want to create a time series of a synthetic precipitation
+field.
+
+We'll start off by creating a Gaussian random field with an exponential
+variogram, which seems to reproduce the spatial correlations of precipitation
+fields quite well. We'll create a daily timeseries over a one dimensional cross
+section of 50km. This workflow is suited for sub daily precipitation time
+series.
+"""
+
+import copy
+import numpy as np
+import matplotlib.pyplot as plt
+import gstools as gs
+
+# fix the seed for reproducibility
+seed = 20170521
```

Comment:
What movie? :wink:

Reply:
Of course Lynch's newest masterpiece Twin Peaks the Return